code/notificationbuilder/src/test/java/com/adobe/marketing/mobile/notificationbuilder/internal/templates/MockInputBoxPushTemplateDataProvider.kt

```diff
@@ -0,0 +1,113 @@
+            PushTemplateConstants.PushPayloadKeys.INPUT_BOX_HINT to MOCKED_HINT,
+            PushTemplateConstants.PushPayloadKeys.INPUT_BOX_FEEDBACK_TEXT to MOCKED_FEEDBACK_TEXT,
+            PushTemplateConstants.PushPayloadKeys.INPUT_BOX_FEEDBACK_IMAGE to MOCKED_FEEDBACK_IMAGE
+        )
+    }
+
+    fun getMockedInputBoxBundleWithRequiredData(): Bundle {
+        val mockBundle = Bundle()
+        mockBundle.putString(PushTemplateConstants.PushPayloadKeys.TITLE, MOCKED_TITLE)
+        mockBundle.putString(PushTemplateConstants.PushPayloadKeys.BODY, MOCKED_BODY)
+        mockBundle.putString(PushTemplateConstants.PushPayloadKeys.VERSION, MOCKED_PAYLOAD_VERSION)
+        mockBundle.putString(PushTemplateConstants.PushPayloadKeys.INPUT_BOX_RECEIVER_NAME, MOCKED_RECEIVER_NAME)
+        mockBundle.putString(PushTemplateConstants.PushPayloadKeys.INPUT_BOX_HINT, MOCKED_HINT)
+        mockBundle.putString(PushTemplateConstants.PushPayloadKeys.INPUT_BOX_FEEDBACK_TEXT, MOCKED_FEEDBACK_TEXT)
+        mockBundle.putString(PushTemplateConstants.PushPayloadKeys.INPUT_BOX_FEEDBACK_IMAGE, MOCKED_FEEDBACK_IMAGE)
+        return mockBundle
+    }
+
+    fun getMockedInputBoxDataMapWithAllKeys(): MutableMap<String, String> {
+        return mutableMapOf(
+            PushTemplateConstants.PushPayloadKeys.TAG to MOCKED_TAG,
+            PushTemplateConstants.PushPayloadKeys.TITLE to MOCKED_TITLE,
+            PushTemplateConstants.PushPayloadKeys.TEMPLATE_TYPE to PushTemplateType.BASIC.value,
+            PushTemplateConstants.PushPayloadKeys.ACTION_URI to MOCKED_ACTION_URI,
+            PushTemplateConstants.PushPayloadKeys.ACTION_TYPE to PushTemplateConstants.ActionType.NONE.name,
+            PushTemplateConstants.PushPayloadKeys.ACTION_BUTTONS to MOCKED_ACTION_BUTTON_DATA,
+            PushTemplateConstants.PushPayloadKeys.BADGE_COUNT to "5",
+            PushTemplateConstants.PushPayloadKeys.BODY to MOCKED_BASIC_TEMPLATE_BODY,
+            PushTemplateConstants.PushPayloadKeys.CHANNEL_ID to MOCKED_CHANNEL_ID,
+            PushTemplateConstants.PushPayloadKeys.EXPANDED_BODY_TEXT to MOCKED_BASIC_TEMPLATE_BODY_EXPANDED,
+            PushTemplateConstants.PushPayloadKeys.BODY_TEXT_COLOR to "FFD966",
+            PushTemplateConstants.PushPayloadKeys.IMAGE_URL to MOCKED_IMAGE_URI,
+            PushTemplateConstants.PushPayloadKeys.LARGE_ICON to MOCKED_LARGE_ICON,
+            PushTemplateConstants.PushPayloadKeys.BACKGROUND_COLOR to "FFD966",
+            PushTemplateConstants.PushPayloadKeys.PRIORITY to MOCKED_PRIORITY,
+            PushTemplateConstants.PushPayloadKeys.VISIBILITY to MOCKED_VISIBILITY,
+            PushTemplateConstants.PushPayloadKeys.REMIND_LATER_TEXT to MOCK_REMIND_LATER_TEXT,
+            PushTemplateConstants.PushPayloadKeys.REMIND_LATER_TIMESTAMP to MOCK_REMIND_LATER_TIME,
+            PushTemplateConstants.PushPayloadKeys.REMIND_LATER_DURATION to MOCK_REMIND_LATER_DURATION,
```

Comment:
Remind later is not supported with Input box template. Please remove keys in L61-L63

Reply:
Same comment for `getMockedInputBoxBundleWithAllKeys`

Reply:
done, thanks